docker/requirements.txt

```diff
@@ -51,3 +51,4 @@ uvloop==0.14.0
 Werkzeug==0.15.4
 wheel>=0.31.0
 zulip==0.6.3
+six>1.14.0
```

Comment:
I think other PRs are running into this problem. Where is six used? Why was this change necessary?

Reply:
looks like it's used in google cloud, I think the problem was because pip changed something 
`ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.
We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

pytest-astropy 0.8.0 requires pytest-cov>=2.0, which is not installed.
pytest-astropy 0.8.0 requires pytest-filter-subpackage>=0.1, which is not installed.
astroid 2.3.3 requires typed-ast<1.5,>=1.4.0; implementation_name == "cpython" and python_version < "3.8", which is not installed.`

Reply:
Nice hunting, this is a good fix. I'm working on a more thorough fix but it's been caught up in lots of silly details.